server/models/user.js

```diff
@@ -81,7 +81,7 @@ export default (sequelize, DataTypes) => {
       foreignKey: 'userId',
       as: 'likes'
     });
-    User.hasOne(models.Profile, {
+    User.hasMany(models.Profile, {
```

Comment:
I changed this relationship to hasMany because I couldn't get sourceKey to work using hasOne. I searched and saw that hasOne does not support sourceKey